packages/frameworks/src/frameworks.ts

```diff
@@ -262,6 +263,71 @@ export const frameworks = [
       },
     ],
   },
+  {
+    name: 'Astro',
+    slug: 'Astro',
+    demo: 'https://astro-template.vercel.app',
+    logo: 'https://raw.githubusercontent.com/vercel/vercel/main/packages/frameworks/logos/astro.svg',
```

Comment:
I see your logo has a dark form with no outline or background — On a dark-themed vercel.com it may show up over a dark background in some cases.

If you have a logo optimized for display on black, consider adding a `darkModeLogo` attribute!

Reply:
I hadn't noticed the `darkModeLogo` prop, thanks for catching this!  Just added the logo we use for dark backgrounds